frontend/src/metabase/reference/segments/SegmentDetail.jsx

```diff
@@ -180,7 +181,18 @@ const SegmentDetail = props => {
       >
         {() => (
           <div className={CS.wrapper}>
-            <div className="pl4 pr3 pt4 mb4 mb1 bg-white rounded bordered">
+            <div
+              className={cx(
+                CS.pl4,
+                CS.pr3,
+                CS.pt4,
+                CS.mb4,
+                CS.mb1,
```

Comment:
I know there's duplicate classes here (`mb4` and `mb1`) for some of these changes but I don't want to make more changes than are already here. We can remove the duplication in another specific PR